CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ download_project(PROJ                googletest
 # when building with Visual Studio
 set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
 
-add_subdirectory(${googletest_SOURCE_DIR} ${googletest_BINARY_DIR})
+add_subdirectory(${googletest_SOURCE_DIR} ${googletest_BINARY_DIR} EXCLUDE_FROM_ALL)
```

Comment:
Sorry, I should have been clearer. My previous comment "I want to keep the example as simple as possible" was meant to imply that I'd prefer not to add the logic you just mentioned. I think the test for whether using the Xcode generator would be a distraction for this particular example. Definitely okay to do it for a real-world project though (some of the projects I use at work do have something like that). Thanks for the proposal anyway.